### DISCUSSION TITLE
Compatability between RabbitMQ Client and Server

### DISCUSSION THREAD

**Discussion by zsharon75** [rabbitmq/rabbitmq-dotnet-client#1534] [Q&A] [answered]
How can I know the compatability between RabbitMQ Client and Server (i.e. which version of RabbitMQ Server the specific .NET RabbitMQ client support)?

**Comment by lukebakken** [comment, marked as answer]
Hello, thanks for using RabbitMQ and this client library.

The AMQP 0.9.1 protocol never changes, which means that pretty much all versions of this library are compatible with any RabbitMQ version.

Having said that, we only support the 6.x versions of this library. You can see which RabbitMQ versions are supported here - https://www.rabbitmq.com/release-information

**Reply by michaelklishin** [reply]
With all RabbitMQ versions after 2.0, 1.x did not implement AMQP 0-9-1 but only AMQP 0-8-0 and 0-9-0. But hopefully there are next to no 1.x installations out in the wild by now.
